CRM/Admin/Form/PaymentProcessor.php

```diff
@@ -200,6 +200,14 @@ public function buildQuickForm($check = FALSE) {
       array('' => ts('- select -')) + $financialAccount,
       TRUE
     );
+    $this->addSelect('payment_instrument_id',
```

Comment:
I think our scoring system is off.

BTW did you mean to disable the 'placeholder' for this field? Normally you'd do that by making it required. Otherwise you get a non-required field that can't be un-selected.